source/community/reactnative/src/adaptive-card.js

```diff
@@ -166,10 +164,17 @@ export default class AdaptiveCard extends React.Component {
 	}
 
 	getAdaptiveCardContent() {
+		let containerStyles = [styles.container]
+		//If contentSize is passed by the user from adaptive card via props, we will set this as height
+		this.props.contentSize && containerStyles.push({ height: this.props.contentSize })
```

Comment:
When we say `size`, it would imply both width and height. If the prop `contentSize` is used only for height why can't we rename the prop as `contentHeight` ?

Reply:
Resolved